modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -227,20 +244,38 @@ const RichText = ({
 	}
 
 	const resetTranslation = useCallback(() => {
-		editorRef.current.editor.setData(currentValue[defaultLocale.localeId]);
-	}, [editorRef, currentValue, defaultLocale]);
+		const data = currentValue[defaultLocale.localeId];
+
+		if (Liferay.FeatureFlags['LPD-11235']) {
+			setCKEditor5Config({
+				...ckEditor5Config,
+				initialData: data ?? '',
+			});
+		}
+		else {
+			editorRef.current.editor.setData(data);
+		}
+	}, [ckEditor5Config, currentValue, defaultLocale, editorRef]);
```

Comment:
I believe there is a bug in resetting of translation, in `master`. Based on the confirmation warning in UI (see video), content should reset to content on default locale. What actually happens is that content is set to blank. In code, `currentValue[defaultLocale.localeId]` returns blank. 

It seems that fixing this requires a bit of restructuring of data structures in this file, which might be a rabbit hole. I'd say this is outside of scope of this task. Because of this, I did not include reset translation test in this PR.

Reply:
This is a business rule exclusive to web content, and is really outside the scope of this task. But just out of curiosity, I believe that if we destructure `localizedValue` from `otherProps` and use it instead of values ​​we will get the expected result.